build.gradle

```diff
@@ -9,6 +17,9 @@ apply plugin: 'maven'
 apply plugin: 'maven-publish'
 apply plugin: 'antlr'
 
+apply plugin: 'osgi'
```

Comment:
Good question .. honestly, I took the configuration from graphql-java-annotations and graphql-java-servlet which is working well. Usually you should choose between the 2 plugins - both actually extend the manifest and the jar with in a different way. But in our case osgi plugin is enough, and it's more clear to use only one. I will remove the bnd plugin. Thank you !